terraform/variables.tf

```diff
@@ -0,0 +1,76 @@
+variable "AWS_REGION" {
+  type = string
+  default = "eu-west-2"
+  
+}
+
+variable "AWS_ACCESS_KEY" {
+  type = string
+  
+}
+
+variable "AWS_SECRET_KEY" {
+  type = string
+  
+}
```

Comment:
We previously had problems with uploading these variables as env variables so keep an eye out for this error in future